docs/tutorial/using-multiple-themes-together.md

```diff
@@ -2,4 +2,481 @@
 title: Using Multiple Themes Together
 ---
 
-_This tutorial is coming soon. In the meantime, check out the [introduction to using multiple themes in the docs](/docs/themes/using-multiple-gatsby-themes)._
+## What this tutorial covers
+
+- Composing multiple themes into a final site using [gatsby-theme-blog](/packages/gatsby-theme-blog/), [gatsby-theme-notes](/packages/gatsby-theme-notes/) and [gatsby-mdx-embed](https://gatsby-mdx-embed.netlify.com/) as examples
+- Component shadowing
+- Component shadowing for Theme-UI theme file
+
+## Prerequisites
+
+- [Gatsby fundamentals](/tutorial/#gatsby-fundamentals)
+- [What are Gatsby themes?](/docs/themes/what-are-gatsby-themes/)
+
+## Step 1: Create a new site using the hello world starter
+
+```shell
+gatsby new multiple-themes https://github.com/gatsbyjs/gatsby-starter-hello-world
+```
+
+## Step 2: Install the themes
+
+The first two themes we will compose together are [gatsby-theme-blog](/packages/gatsby-theme-blog/) and [gatsby-theme-notes](/packages/gatsby-theme-notes/).
+
+```shell
+npm install gatsby-theme-blog gatsby-theme-notes
+```
+
+## Step 3: Update gatsby-config.js
+
+The themes need to be added to the plugins array and we need some basic site metadata too.
```

Comment:
```suggestion
The themes need to be added to the plugin array and the site metadata that comes with a starter should be renamed for personalization.
```
In the doc try to avoid using the first person “I”, “we”, “let’s”, and “us” per the [style guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/)